### ISSUE TITLE
Json import not possible anymore in v1.1.5

### ISSUE THREAD

**Issue by Tims777** [denoland/fresh#1106]
The recent dependency update seems to have caused a regression when importing `json` files.

```
Listening on http://localhost:8000/
X [ERROR] The file "static/world.json" was loaded with the "js" loader

    islands/Globe.tsx:4:33:
      4 │ import { default as world } from "../static/world.json" assert { type: "json" };
        ╵                                  ~~~~~~~~~~~~~~~~~~~~~~

  This import assertion requires the loader to be "json" instead:

    islands/Globe.tsx:4:65:
      4 │ import { default as world } from "../static/world.json" assert { type: "json" };
        ╵                                                                  ~~~~~~~~~~~~

  You need to either reconfigure esbuild to ensure that the loader for this file is "json" or you
  need to remove this import assertion.

An error occurred during route handling or page rendering. Error: Build failed with 1 error:
islands/Globe.tsx:4:33: ERROR: The file "static/world.json" was loaded with the "js" loader
    at failureErrorWithLog (https://deno.land/x/esbuild@v0.17.11/mod.js:1612:15)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1024:25
    at runOnEndCallbacks (https://deno.land/x/esbuild@v0.17.11/mod.js:1447:45)
    at buildResponseToResult (https://deno.land/x/esbuild@v0.17.11/mod.js:1022:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1051:16
    at responseCallbacks.<computed> (https://deno.land/x/esbuild@v0.17.11/mod.js:673:9)
    at handleIncomingPacket (https://deno.land/x/esbuild@v0.17.11/mod.js:728:9)
    at readFromStdout (https://deno.land/x/esbuild@v0.17.11/mod.js:649:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1898:11
 ```
 
 Using fresh v1.1.4 instead of v1.1.5 works as expected.

**Comment by dannygim** [comment]
I have the same problem.
Is it caused by the updated `esbuild` to v0.17.11?

**Comment by kt3k** [comment]
probably related to: https://github.com/lucacasonato/esbuild_deno_loader/issues/46

**Comment by lucacasonato** [comment]
https://github.com/lucacasonato/esbuild_deno_loader/issues/46 is fixed. Will be part of Fresh 1.2.0 next week.

**Comment by marvinhagemeister** [comment]
We just released Fresh `v1.2.0`. With that json imports work again 👍
